File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/asynchronousNotificationsofChanges/IAsynchronousNotificationsOfChanges.java
package org.ngbp.jsonrpc4jtestharness.rpc.asynchronousNotificationsofChanges;

import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcMethod;
import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcType;

@JsonRpcType
public interface IAsynchronousNotificationsOfChanges {

    @JsonRpcMethod("org.atsc.notify")
    Object contentAdvisoryRatingChangeNotification();

}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/asynchronousNotificationsofChanges/IAsynchronousNotificationsOfChanges.kt
package org.ngbp.jsonrpc4jtestharness.rpc.asynchronousNotificationsofChanges

import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcMethod
import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcType
import org.ngbp.jsonrpc4jtestharness.rpc.RpcResponse

@JsonRpcType("")
interface IAsynchronousNotificationsOfChanges {
    @JsonRpcMethod("org.atsc.notify")
    fun contentAdvisoryRatingChangeNotification(): RpcResponse
}